core/ErrorFlusher.h

```diff
@@ -10,9 +10,8 @@ namespace core {
 
 class ErrorFlusher {
 public:
-    virtual void flushErrors(spdlog::logger &logger, std::vector<std::unique_ptr<ErrorQueueMessage>> error) = 0;
-    virtual void flushErrorCount(spdlog::logger &logger, int count) = 0;
-    virtual void flushAutocorrects(const GlobalState &gs, FileSystem &fs) = 0;
+    virtual void flushErrors(spdlog::logger &logger, std::vector<std::unique_ptr<ErrorQueueMessage>> errors,
```

Comment:
(Optional) WDYT about changing the order of arguments to this to put the more globalish things first? Normally, `gs` is near the front of the list, and `file` should probably go before `errors`.

`logger`, `gs`, `file`, `errors`